fe/fe-core/src/main/java/com/starrocks/authorization/TablePEntryObject.java

```diff
@@ -97,6 +97,13 @@ public static TablePEntryObject generate(List<String> tokens) throws PrivilegeEx
                 throw new PrivObjNotFoundException("cannot find catalog: " + catalogName);
             }
             catalogId = catalog.getId();
+            // As a system database, `information_schema` is case-insensitive
+            // across both internal and external catalogs. Queries can access it
+            // regardless of letter case (e.g. `information_schema`, `INFORMATION_SCHEMA`, `Information_Schema`).
+            if (tokens.size() == 2 && "information_schema".equalsIgnoreCase(tokens.get(0))) {
```

Comment:
Can we consider unifying the logical differences between the outer table and the inner table at the match position?